docs/content/docs/dev/table/sqlClient.md

```diff
@@ -601,6 +601,24 @@ This configuration:
 
 <span class="label label-danger">Attention</span> Compared to the interactive mode, SQL Client will stop execution and exits when there are errors.
 
+### Deploy SQL Files in an Application Cluster
+
+SQL Client also supports to deploy a SQL script file to an application cluster with the `-f` option if you specify the deployment target in the config.yaml or startup options.
+Here is an example to deploy script file in an application cluster.
```

Comment:
Application Cluster is sometimes capitalised and sometimes not - I suggest using the capitalised form consistently